tests/README.md

```diff
@@ -1,17 +1,17 @@
 
 ## Preparations
 
-1. The following executables must be copied or generated or linked into these locations, `mydumper` and `sync_diff_inspector` can be downloaded from [tidb-enterprise-tools-latest-linux-amd64](http://download.pingcap.org/tidb-enterprise-tools-latest-linux-amd64.tar.gz):
+1. The following executables must be copied or generated or linked into these locations, `mydumper` and `sync_diff_inspector` can be downloaded from [tidb-enterprise-tools-latest-linux-amd64](http://download.pingcap.org/tidb-enterprise-tools-latest-linux-amd64.tar.gz), `tidb-server` can be downloaded from [tidb-master-linux-amd64](https://download.pingcap.org/tidb-master-linux-amd64.tar.gz):
```

Comment:
In DM 2.0, we do not need `mydumper` now, how about removing it from this README and do not check where it exsits? cc @lichunzhu 

(but when cherry-pick to 1.0, we still needed it).

Reply:
Yes, I think we should remove it for master, v2.0 but kept it for v1.0.

Reply:
Shall we remove `mydumper-path` config in tests/*/dm-task.yaml ?

Reply:
Yes, you can remove them.

Reply:
How about remove them in another pr?

Reply:
OK